### ISSUE TITLE
Windows: set PRISM_PATH in the (x)prism.bat files during installation

### ISSUE THREAD

**Issue by kleinj** [prismmodelchecker/prism#43]
Currently, in the prism.bat (and xprism.bat), the location of the PRISM directory is hard-coded as `..`:
https://github.com/prismmodelchecker/prism/blob/03d83d4abb5a5a1047863f5afab5db4e8596bd2e/prism/src/bin/prism.bat.win#L5-L10

This should probably be replaced by the actual installation location during the installation of the binary with the installer. There seems to be a way to replace the corresponding lines via some tweaking of the NSIS installer configuration:

http://nsis.sourceforge.net/ReplaceInFile
http://nsis.sourceforge.net/Replacing_Lines_in_a_Text_File

**Comment by davexparker** [comment]
Yes, good idea. Never got around to seeing if NSIS could do that.
